Makefile

```diff
@@ -9,7 +9,7 @@ swagger:
 	@$(GOPATH)/bin/swag init -d "./" -g "apis/configwatch.go"
 
 build:
-	@go build
+	@CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-extldflags=-static"
```

Comment:
I think we can drop the GOOS and GOARCH lines as it should pick up the correct defaults.  I would also drop 'ldflags'.  We should only need 'CGO_ENABLED=0 go build' to build static binaries or 'go build -tags osusergo,netgo'